staging/src/k8s.io/kubectl/pkg/cmd/annotate/annotate.go

```diff
@@ -304,15 +317,21 @@ func (o AnnotateOptions) RunAnnotate() error {
 			}
 			name, namespace := info.Name, info.Namespace
 
+			accessor, err := meta.Accessor(obj)
```

Comment:
What will this return if `o.resourceVersion` is equal to 0?

Reply:
If len(o.resourceVersion) == 0, will also return obj with empty resourceVersion.
Added test case of this.